Formula/w/weaviate.rb

```diff
@@ -0,0 +1,18 @@
+class Weaviate < Formula
+  desc "Open-source vector database that stores both objects and vectors"
+  homepage "https://weaviate.io/developers/weaviate/"
+  url "https://github.com/weaviate/weaviate/archive/refs/tags/v1.27.6.tar.gz"
+  sha256 "d8cad0339ccfd081be6afe1afd879f64bba224ed7ecbdb8b974853b536584330"
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "./cmd/weaviate-server"
+    bin.install "weaviate-server" => "weaviate"
+  end
+
+  test do
+    assert_match '"version":"1.27.6"', shell_output("#{HOMEBREW_PREFIX}/bin/weaviate 2>&1", 1)
```

Comment:
```suggestion
    assert_match version.to_s, shell_output("#{bin}/weaviate --version")
```

Reply:
Done

Reply:
Somehow, brew audit is failing if I specify "#{bin}" instead of bin/"weaviate", am I missing something?